app/test_main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+import pytest
+from unittest import mock
+from app.main import can_access_google_page
```

Comment:
![image](https://user-images.githubusercontent.com/104218550/221212960-b825bb05-7d27-476c-b047-29d6d82448c3.png)

Reply:
still not fixed

Reply:
i hope, i finally made it right